.gitlab-ci/lint.yml

```diff
@@ -1,13 +1,24 @@
 ---
-yamllint:
-  extends: .job
-  stage: unit-tests
-  tags: [light]
-  variables:
-    LANG: C.UTF-8
-  script:
-    - yamllint --strict .
-  except: ['triggers', 'master']
+generate-pre-commit:
+  image: 'mikefarah/yq@sha256:bcb889a1f9bdb0613c8a054542d02360c2b1b35521041be3e1bd8fbd0534d411'
```

Comment:
HI @VannTen 

 To make the image more readable, is it ok to change the `mikefarah/yq@sha256:bcb889a1f9bdb0613c8a054542d02360c2b1b35521041be3e1bd8fbd0534d411`
to `mikefarah/yq:4.44.1-githubaction`